docs/specs/output.yml

```diff
@@ -21,6 +21,15 @@ outputs:
       ]
     }
 ---
+# Publish manifest contains config baseUrl
+inputs:
+  docfx.yml:
+outputs:
+  .publish.json: |
+    {
+      "base_url": "https://docs.com"
```

Comment:
nit: is there a test case for these additional properties in .publish.json like name.